samples/client/petstore/php/SwaggerClient-php/lib/ObjectSerializer.php

```diff
@@ -162,6 +162,37 @@ public function toString($value)
     }
 
     /**
+     * Serialize an array to a string.
+     *
+     * @param array  $collection       collection to serialize to a string
+     * @param string $collectionFormat the format use for serialization (csv,
+     * ssv, tsv, pipes, multi)
+     *
+     * @return string
+     */
+    public function serializeCollection(array $collection, $collectionFormat, $allowCollectionFormatMulti=false)
+    {
+        if ($allowCollectionFormatMulti && ('multi' === $collectionFormat)) {
+            return http_build_query($collection, null, '&');
```

Comment:
Yes, you're right, @xhh, I haven't account for multidimensional arrays. That's a bug.

I'll look into -- either I have to do query string building myself or I'll postprocess the result of `http_build_query()` and weed out the encoded `[]`.